cnf-tests/Dockerfile.openshift

```diff
@@ -9,7 +9,7 @@ RUN mkdir -p $PKG_PATH
 COPY . $PKG_PATH/
 WORKDIR $PKG_PATH
 
-RUN make build-bin
+RUN make test-bin
 
 WORKDIR /
```

Comment:
why we have this line here?

Reply:
You must really hate this. You made the same question before :-)
https://github.com/openshift-kni/cnf-features-deploy/pull/138#discussion_r409425961

I took inspiration from other dockerfiles, I guess it's to restore WORKDIR to what it was before.

Reply:
yeah but then we jump to another container this is only the builder container

Reply:
removed it